docs/ops/pooling/AdaptiveAvgPool_8.md

```diff
@@ -0,0 +1,73 @@
+## AdaptiveAvgPool<a name="AdaptiveAvgPool"></a> {#openvino_docs_ops_pooling_AdaptiveAvgPool_8}
+
+**Versioned name**: *AdaptiveAvgPool-8*
+
+**Category**: *Pooling*
+
+**Short description**: Applies average pooling with adaptive kernel size over the input.
+
+**Detailed description**: This operation calculates the output based on the input and `output_size`.
+The kernel dimensions are calculated using the following formulae for the `NCDHW` input case:
+
+\f[
+\begin{array}{lcl}
+d_{start}&=&floor(i*D_{in}/D_{out})\\
+d_{end}&=&ceil((i+1)*D_{in}/D_{out})\\
+h_{start}&=&floor(j*H_{in}/H_{out})\\
+h_{end}&=&ceil((j+1)*H_{in}/H_{out})\\
+w_{start}&=&floor(k*W_{in}/W_{out})\\
+w_{end}&=&ceil((k+1)*W_{in}/W_{out})
+\end{array}
+\f]
+
+The output is calculated following this formula:
+
+\f[
+Output(i,j,k)=\frac{Input[d_{start}:d_{end},h_{start}:h_{end},w_{start}:w_{end}]}{(d_{end}-d_{start})*(h_{end}-h_{start})*(w_{end}-w_{start})}
+\f]
+
+**Attributes**:
+
+* *output_size*
+
+  * **Description**: the target spacial output shape `[W_out]` for 3D input, `[H_out,W_out]` for 4D input,
+    `[D_out,H_out,W_out]` for 5D input.
+  * **Range of values**: positive integer values
```

Comment:
Maybe we should have not attribute `output_size`, but the input `output_size`?

Reply:
We should discuss this I think because `output_size` is an attribute in all frameworks which have adaptive pooling.

Reply:
It seems this is an input in TensorFlow. Please double-check it.